src/http/invoke-http/http/_req-fmt.js

```diff
@@ -47,6 +47,11 @@ module.exports = function requestFormatter ({ verb, route, req, $default }) {
 
   // Context, which now contains the HTTP method
   request.requestContext = {
+    authorizer: {
```

Comment:
This would not be part of everyone's API Gateway config, and thus should only be added conditionally to the requestContext. Also, need to add conditionally to the rest code path for `HTTP` APIs in v1.0 mode.